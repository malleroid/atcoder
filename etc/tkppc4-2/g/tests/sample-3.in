4 300
4089
2800 20 30 40
3200 30 40 50
3600 30 50 60
4000 20 70 80
