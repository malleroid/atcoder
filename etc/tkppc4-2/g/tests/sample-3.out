4089.00000
